tests/comdb2_unused_files.test/runit

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+bash -n "$0" | exit 1
+
+. ${TESTSROOTDIR}/tools/cluster_utils.sh
+. ${TESTSROOTDIR}/tools/runit_common.sh
+
+export debug=1
+[[ "$debug" != "0" ]] && set -x
+
+function run_test
+{
+    typeset -l j=0
+    typeset -l master=$(get_master)
+    $CDB2SQL_EXE $CDB2_OPTIONS $DBNAME default "create table t1 (a int)"
+
+    while [[ "$j" -lt 10 ]] ; do
+        $CDB2SQL_EXE $CDB2_OPTIONS $DBNAME default "truncate t1"
+        let j=j+1
+    done
+
+    sleep 10
```

Comment:
do the files get added to unused_files asynchronously or immediately after the table has been truncated?

Reply:
👍